spec/Section 4 -- Introspection.md

```diff
@@ -242,6 +269,8 @@ Fields
 * `kind` must return `__TypeKind.SCALAR`.
 * `name` must return a String.
 * `description` may return a String or {null}.
+* `punctuatedName` must return `name`.
+* `namedType` must return a reference to the __Type object {this}.
```

Comment:
`{this}` doesn't have any specific meaning in the GraphQL spec - we must be careful not to use JavaScript idioms.

```suggestion
* `punctuatedName` must return the same value as the `name` field would.
* `namedType` must return this type.
```